ReSwift-ThunkTests/ExpectThunk.swift

```diff
@@ -8,7 +8,6 @@
 
 import XCTest
 import ReSwift
-import ReSwiftThunk
```

Comment:
ahhh, it's a warning to leave this in when integrating into a 3rd party. I will think about it what to do here